worlds/AutoWorld.py

```diff
@@ -118,6 +120,29 @@ def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> Aut
         return new_class
 
 
+class WebWorldRegister(type):
+    def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> WebWorldRegister:
+        # don't allow an option to appear in multiple groups, allow "Item & Location Options" to appear anywhere by the
+        # dev, putting it at the end if they don't define options in it
+        option_groups: List[OptionGroup] = dct.get("option_groups", [])
+        item_and_loc_options = [LocalItems, NonLocalItems, StartInventory, StartInventoryPool, StartHints,
+                                StartLocationHints, ExcludeLocations, PriorityLocations, ItemLinks]
+        seen_options = []
+        item_group_in_list = False
+        for group in option_groups:
+            assert group.name != "Game Options", "Game Options is a pre-determined group and can not be defined."
+            if group.name == "Item & Location Options":
+                group.options.extend(item_and_loc_options)
+                item_group_in_list = True
```

Comment:
i didn't notice this before (and it's my bug) but you can put one of the item_and_loc_options in a different group and it won't render correctly. possible solution
```suggestion
            if group.name == "Item & Location Options":
                group.options.extend(item_and_loc_options)
                item_group_in_list = True
            else:
                for option in group.options:
                    assert option not in item_and_loc_options, f"{option} cannot be moved out of the \"Item & Location Options\" Group"
```